src/resultsdisassemblypage.cpp

```diff
@@ -199,3 +201,46 @@ void ResultsDisassemblyPage::setData(const Data::DisassemblyResult &data)
         m_objdump = QLatin1String("aarch64-linux-gnu-objdump");
     }
 }
+
+void ResultsDisassemblyPage::resetCallStack()
+{
+    m_callStack.clear();
+}
+
+void calcFunction(QString asmLineCall, QString* offset, QString* symName)
+{
+    QStringList sym = asmLineCall.trimmed().split(QLatin1Char('<'));
+    *offset = sym[0].trimmed();
```

Comment:
will crash if the line is empty, use .value()

Reply:
I have fixed this in one of next commits.